modules/reactor/test/tests/meshgenerators/core_mesh_generator/gold/core_hex_2d_datadriven_out.json

```diff
@@ -0,0 +1,490 @@
+{
+    "app_name": "main",
+    "current_time": "Thu Dec 14 17:01:57 2023",
+    "executable": "/Users/shikharkumar/Documents/Codes/moose/modules/reactor/reactor-opt",
+    "executable_time": "Thu Dec 14 16:47:00 2023",
+    "libmesh_version": "",
```

Comment:
I think there s a reporter parameter to ditch this extra infomration from the output

Reply:
I'm not sure which parameter this is,  can you point me to what would be controlling this?

https://mooseframework.inl.gov/source/reporters/MeshMetaDataReporter.html

Reply:
```
[Outputs]
  [out]
    type = JSON
    execute_on = FINAL
    execute_system_information_on = none.  <- that one
  []
[]
```

Reply:
not a reporter param I guess

Reply:
Thanks